src/chat-bubble/chat-bubble-in.ts

```diff
@@ -0,0 +1,72 @@
+import { html, LitElement, nothing } from "lit";
+import { property } from "lit/decorators.js";
+import {
+  BaseSlots,
+  InParts,
+  STATUS_TO_ICON_MAP,
+  STATUS_TO_LOCALE_MAP,
+  type STATES,
+} from "./constants";
+
+export class ChatBubbleIn extends LitElement {
+  @property({ type: String, reflect: true })
+  public timestamp!: string;
+
+  @property({ type: String, reflect: true })
+  public status?: (typeof STATES)[number];
+
+  @property({ type: Boolean, reflect: true, attribute: "fully-rounded" })
+  public fullyRounded: boolean = false;
+
+  private _renderFailureIndicator() {
+    if (this.status !== "failed") return nothing;
+
+    const icon = STATUS_TO_ICON_MAP.failed;
+
+    return html`
+      <div
+        class=${InParts.FAILURE_INDICATOR}
+        part=${InParts.FAILURE_INDICATOR}
+      >
+        ${icon}
+      </div>
+    `;
+  }
+
+  private _renderStatus() {
+    if (!this.status || this.status === "failed") return nothing;
+
+    const stateMessage = STATUS_TO_LOCALE_MAP[this.status];
+    const icon = STATUS_TO_ICON_MAP[this.status];
+
+    return html`
+      <div
+        slot=${BaseSlots.FOOTER}
+        class=${InParts.STATUS}
+        part=${InParts.STATUS}
+      >
+        ${icon}
+        <span>${stateMessage}</span>
+      </div>
+    `;
+  }
+
+  protected override render() {
+    return html`
+      <div
+        class=${InParts.ROOT}
+        part=${InParts.ROOT}
+      >
+        ${this._renderFailureIndicator()}
+        <tap-chat-bubble-base
+          author="in"
+          ?fully-rounded=${this.fullyRounded}
+          timestamp=${this.timestamp}
+        >
+          <slot slot=${BaseSlots.BODY}></slot>
+          ${this._renderStatus()}
+        </tap-chat-bubble-base>
+      </div>
+    `;
+  }
+}
```

Comment:
Same feedback as for ChatBubbleOut. My main concern is the redefinition of the same properties. I’m not opposed to creating separate components for each type (in and out)—in fact, I think that’s a good pattern. However, we should avoid duplicating properties unnecessarily.